src/Apps/Artwork/Components/ArtworkLightboxPlaceholder.tsx

```diff
@@ -44,6 +44,7 @@ export const ArtworkLightboxPlaceholder: React.FC<ArtworkLightboxPlaceholderProp
           height="100%"
           alt="Loading"
           lazyLoad={lazyLoad}
+          style={{ filter: "blur(10px)" }}
```

Comment:
@joeyAghion - check out the comment in this file. I think its safe to keep this since we're now disabling lazyload here. It looks a lot better with it on.